google/cloud/security/notifier/pipelines/email.py

```diff
@@ -0,0 +1,62 @@
+
+# Copyright 2017 Google Inc.
```

Comment:
We might need to revisit this copyright, to be less google-centric for major contributions.  :)  Michael has a plan for this.  @mirons-google